torch/csrc/jit/codegen/cuda/lower_scalar_hoist.cpp

```diff
@@ -186,12 +187,32 @@ std::pair<Val*, bool> CommonScalarMap::hoistScalarImpl(
   return {value, has_tensor_dependency};
 }
 
+namespace {
+
+std::list<ValInfo> getVariablesFromLoops(
```

Comment:
nit: `getLoopIndices`

Reply:
I will throw your reviews to https://github.com/csarofeen/pytorch/pull/2257